src/libstd/sys/unix/process/process_unix.rs

```diff
@@ -52,11 +53,12 @@ impl Command {
                     drop(input);
                     let Err(err) = self.do_exec(theirs, envp.as_ref());
                     let errno = err.raw_os_error().unwrap_or(libc::EINVAL) as u32;
+                    let errno = errno.to_be_bytes();
                     let bytes = [
-                        (errno >> 24) as u8,
-                        (errno >> 16) as u8,
-                        (errno >> 8) as u8,
-                        (errno >> 0) as u8,
+                        errno[0],
+                        errno[1],
+                        errno[2],
+                        errno[3],
```

Comment:
Uhm... with this code, the `errno` is always encoded in big endian into the array. But before, it was always swapped -- even on big endian devices -- right? And I don't see any `#[cfg(target_endianess = ...)]` attributes anywhere. So your PR changes behavior, right? Which means either your change or the previous code is buggy? Is this code tested on big endian CPUs?

I unfortunately don't know a lot about these unix system APIs. Could you maybe link me some resource which specifies that this has to be big endian? Or is this *just* used internally? Like... the data is used below. But if it's just internally, why swap any bytes at all? Why not keep it in native endianess?